elements/lisk-dpos/test/unit/apply.spec.ts

```diff
@@ -40,6 +39,10 @@ import {
 } from '../../src/constants';
 import { StateStoreMock } from '../utils/state_store_mock';
 
+const MS_IN_A_SEC = 1000;
+const GENESIS_BLOCK_TIMESTAMP =
+	new Date(Date.UTC(2020, 5, 15, 0, 0, 0, 0)).getTime() / MS_IN_A_SEC;
+
```

Comment:
I saw these multiple times in `lisk-dpos` tests. Can we define it somewhere shared?